pkg/testutil/fakes.go

```diff
@@ -782,3 +807,586 @@ func (s *FakeSecrets) Delete(ns, name string) error {
 func (s *FakeSecrets) GetVmiPassword() (string, error) {
 	return monitoring.GetVmiPassword(s)
 }
+
+// ----- simpleCohClusterLister
+// simpleCohClusterLister implements the CohClusterLister interface.
+type simpleCohClusterLister struct {
+	kubeClient      kubernetes.Interface
+	cohOprClientSet cohoprclientset.Interface
+}
+
+// List lists all CohClusters.
+func (s *simpleCohClusterLister) List(selector labels.Selector) (ret []*cohv1beta1.CohCluster, err error) {
```

Comment:
using ret for the named return value doesn't really help since we explicitly return values and that name doesn't tell us much.  If we want to use it for doc purposes then user a more descriptive name or else get rid of the named returns.  Named returns are not used consistently in this file.

Reply:
@pfmackin, thanks for reviewing.

The names were used here because this implements an interface function that names them ...
```
// CohClusterLister helps list CohClusters.
type CohClusterLister interface {
	// List lists all CohClusters in the indexer.
	List(selector labels.Selector) (ret []*v1beta1.CohCluster, err error)
	// CohClusters returns an object that can list and get CohClusters.
	CohClusters(namespace string) CohClusterNamespaceLister
	CohClusterListerExpansion
}
```
If you think that I should get rid of them, I will.

Reply:
Thanks, I got rid of all the unnecessary named returns in this file.